### PR TITLE
Update wesnoth-ala.po

### PR DESCRIPTION
**Pull request by MultiSeptim**
Next turn @celerini... And maybe the final.

### CODE REVIEW

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 1]
```diff
@@ -2444,6 +2422,8 @@ msgid ""
 "But you put the sticks between my wheels twice, first when you defeated "
 "Grunto and his human ally Dadoc, and then you started to go to the east."
 msgstr ""
+"Ale dvakrát jsi mi vrazil klacek mezi kola. Nejdřív, když jsi porazil Grunta "
+"a jeho lidského spojence Dadoka, a pak, když si se vydal na východ."
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"a jeho lidského spojence Dadoka, a pak, když ses vydal na východ."
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 2]
```diff
@@ -2467,13 +2451,20 @@ msgid ""
 "stealthily, to take all these territories, and my tribe will become the most "
 "powerful tribe of orcs on the whole continent!"
 msgstr ""
+"Dobře vím, že se budou zlobit za zničení Amoranu. Ale se okamžitě stáhnu, "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"Dobře vím, že se budou zlobit za zničení Amoranu. Ale okamžitě se stáhnu, "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 3]
```diff
@@ -3272,6 +3269,12 @@ msgid ""
 "ready to hover in the sky and put his very sharp sight at disposal to find "
 "what or who had been missing."
 msgstr ""
+"Divaid a Ranfy se vrátili do svých lesů, brblajíc, že pomohli západním "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"Divaid a Ranfy se vrátili do svých lesů, brblajíce, že pomohli západním "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 4]
```diff
@@ -3272,6 +3269,12 @@ msgid ""
...
+"wesnothským neurvalcům se vyhrabat z bídy. Ale ve skutečnosti to bylo jen "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"wesnothským neurvalcům vyhrabat se z bídy. Ale ve skutečnosti to bylo jen "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 5]
```diff
@@ -3272,6 +3269,12 @@ msgid ""
...
+"takové mumlání pod fousy pro zachování formy. Protože když bylo něco "
+"ztraceno nebo se někdo ztratil v těch neznámých územích, Ranfy byl vždy "
+"připraven vznést se k obloze a dát k dispozici svůj velmi ostrý zrak, aby "
+"našel, to, co bylo ztraceno nebo toho, kdo byl nezvěstný."
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]: the first comma could remain, but I think it is better without it
```suggestion
"našel to, co bylo ztraceno, nebo toho, kdo byl nezvěstný."
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 6]
```diff
@@ -3293,6 +3300,12 @@ msgid ""
 "but rather the best sentries that Amoran could have, and soon the gryphons "
 "were considered citizens of Amoran like everyone else."
 msgstr ""
+"Ranfy zmizel hned poté, co skončila bitva proti Gan-Tarkasovi. O dva měsíce "
+"později se vrátil a ne sám. Udělal si hnízdo na jediné věži, která zbyla ze "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"později se vrátil, a ne sám. Udělal si hnízdo na jediné věži, která zbyla ze "
```

**Review by celerini — CHANGES_REQUESTED** [review]
I am done.

**Comment by MultiSeptim** [comment]
Ready to merge

**Comment by cooljeanius** [comment]
ok merged

**Pull request merged**